Formula/apachetop.rb

```diff
@@ -27,21 +23,20 @@ class Apachetop < Formula
   depends_on "pkg-config" => :build
   depends_on "adns"
   depends_on "ncurses"
-  depends_on "pcre"
+  depends_on "pcre2"
 
   on_linux do
     depends_on "readline"
   end
 
   def install
-    system "./autogen.sh"
-    system "./configure", "--prefix=#{prefix}",
+    ENV.append "CXX", "-std=gnu++17"
+
+    system "./configure", *std_configure_args,
                           "--mandir=#{man}",
-                          "--disable-debug",
-                          "--disable-dependency-tracking",
                           "--with-logfile=/var/log/apache2/access_log",
```

Comment:
Should we set this to at least `#{var}`?

Reply:
yeah, that sounds good to me.

Reply:
👍